SFBaseKitDemo/SFBaseKitDemo/AppCoordinator/AppCoordinator.swift

```diff
@@ -24,11 +23,11 @@ class AppCoordinator: Coordinator {
         self.window?.makeKeyAndVisible()
         
         let loginCoordinator = LoginCoordinator(navigationController: navigationController)
+        loginCoordinator.parentCoordinator = self
```

Comment:
addChildCoordinator automatically sets parentCoordinator. This line is not needed.

Reply:
fixed